mlrun/feature_store/api.py

```diff
@@ -405,7 +408,12 @@ def ingest(
         raise mlrun.errors.MLRunInvalidArgumentError(
             "feature set and source must be specified"
         )
-
+    entities_keys = [e.name for e in featureset.spec.entities]
+    for step_name, step_value in featureset.graph.steps.items():
+        if step_name == get_name(name=None, class_name=DropFeatures):
```

Comment:
Doesn't look like `get_name` is doing anything useful here.
```suggestion
        if step_name == "DropFeatures"
```
or
```suggestion
        if step_name == DropFeatures.__name__
```

Reply:
fixed